src/fable-standalone/src/Interfaces.fs

```diff
@@ -64,5 +57,7 @@ type IFableManager =
     abstract GetDeclarationLocation: parseResults: IParseResults * line: int * col: int * lineText: string -> Async<Range option>
     abstract GetToolTipText: parseResults: IParseResults * line: int * col: int * lineText: string -> Async<string[]>
     abstract GetCompletionsAtLocation: parseResults: IParseResults * line: int * col: int * lineText: string -> Async<Completion[]>
-    abstract CompileToBabelAst: fableLibrary: string * parseResults: IParseResults * fileName: string * ?config: CompilerConfig -> IBabelResult
```

Comment:
@alfonsogarciacaro Are you ok with me reverting this change, can we keep the contract unchanged? So we can pass more compiler parameters to control the compilation, without having to add new optional parameters?

Reply:
Yes, it was just a quick fix. Though I'm trying to avoid records for options as they easily break APIs too. What do you think of an interface with a helper to build it using optional arguments as [in here](https://github.com/fable-compiler/Fable/blob/bacc5e26196b8ad29cc8b5e25129e6fb76bcc832/src/Fable.Transforms/Global/Compiler.fs#L17-L30)?

BTW, in a hacky way I managed to build fable-compiler-js and compile and run the tests, but you're going to need the fixes in this commit: https://github.com/fable-compiler/Fable/commit/bacc5e26196b8ad29cc8b5e25129e6fb76bcc832

The only thing that gave me issues was this line, which I had to comment out: https://github.com/fable-compiler/Fable/blob/bacc5e26196b8ad29cc8b5e25129e6fb76bcc832/tests/Main/StringTests.fs#L10